app/Http/Controllers/Admin/SettingsController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Carbon\Carbon;
+use App\Alexa\Models\Settings;
+use Illuminate\Support\Facades\DB;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Settings\UpdateRequest;
+
+class SettingsController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+        $this->middleware('can.admin');
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function edit()
+    {
+        return view('settings.edit');
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param \App\Http\Requests\Settings\UpdateRequest $request
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function update(UpdateRequest $request)
+    {
+        DB::transaction(function () use ($request) {
+            $updateFields = $request->all();
+            $updateFields['applications_start_at'] = Carbon::parse($updateFields['applications_start_at']);
```

Comment:
`$updateFields['applications_start_at'] = Carbon::parse($request->applications_start_at);`

Why does Carbon parse it and Model does not ?

Reply:
Don't know, without this I get and error `Unexpected data found. Data missing`.
My hint is that when a field is as date it try to convert it by calling `Carbon::createFromFormat()`, and the default format in the input form for the type date doesn't match.

Reply:
What is the format of the input?

Reply:
`Y-m-dTh:i`, I have to reformat it in the view

Reply:
Humm, I will take a look tomorrow on this. Don't worry, we'll make it work without hacks!

Reply:
Going to keep it this way.

Reply:
:+1: